compiler/psi/src/org/jetbrains/kotlin/parsing/KotlinExpressionParsing.java

```diff
@@ -59,17 +59,15 @@ private static ImmutableMap<String, KtToken> tokenSetToMap(TokenSet tokens) {
         return builder.build();
     }
 
-    private static final TokenSet TYPE_ARGUMENT_LIST_INTENTION_INDICATORS = TokenSet.orSet(
-            Precedence.POSTFIX.getOperations(),
-            TokenSet.create(
-                    RBRACKET, RBRACE, RPAR, EQ, COLON, COLONCOLON, SEMICOLON, COMMA,
+    private static final TokenSet TYPE_ARGUMENT_LIST_INTENTION_INDICATORS = TokenSet.create(
+            RBRACKET, RBRACE, RPAR, EQ, COLON, SEMICOLON, COMMA,
 
-                    EQEQEQ, EXCLEQEQEQ, EQEQ, EXCLEQ, SAFE_ACCESS, ELVIS, AS_KEYWORD, IS_KEYWORD, NOT_IS, NOT_IN, RANGE, RANGE_UNTIL,
```

Comment:
In the updated approach it should be a lot clearer. `SAFE_ACCESS` was removed from that list, because it is already contained in `Precedence.POSTFIX.getOperations()`. It is now part of the `STRONG_TYPE_ARGUMENT_LIST_INDICATORS`.